### ISSUE TITLE
[Bug] Visual Disappearing Bug

### ISSUE THREAD

**Issue by mxnmnm** [Tschipp/BuildersBag#12]
**Expected Behavior**
Item doesn't disappear when you hover over it.

**Actual Behavior**
Item disappears until you click on one of the modules or update the bag in some way. I know you saw this already but I figured I should make an issue so it's easy to track.

**Steps to Reproduce**
Hover over an item that's put in the selection slot of the bag and see it disappear

**Version of Minecraft, version of Builder's Bag, version of Addons, version of other Builder's Bag-compatible mods, Single Player or Server**
 1.12.2 and version 1.1.1.11 and I have the project e addon
If you need the versions for other things let me know but I'm to lazy to look for the rest of them but they should be the latest as of 5/16/2020
I do have optifine in case that makes a difference because I know it does weird things with rendering

Here's a video of it in action:
https://youtu.be/s8Be7jHuvJs

**Comment by Tschipp** [comment]
That is weird, definitely doesn't happen on my end. No idea what could be causing this. Anything special in the logs?

**Comment by mxnmnm** [comment]
Not sure what I should be looking for so here's the full log https://gist.github.com/mxnmnm/e2e330616ad58f360ff6062a31f5247d

Just to be sure you want latest.log right?

**Comment by Tschipp** [comment]
Hmm, can't find anything weird there

**Comment by mxnmnm** [comment]
Yea I might test it out later to figure out which mod is causing it because rendering bugs are by far the most common bugs when it comes to big and heavy modpacks

**Comment by Tschipp** [comment]
Yeah totally, also if you have optifine that might also be the issue

**Comment by Tschipp** [comment]
This may be related to #8

**Comment by Tschipp** [comment]
Could you try with the most recent release?

**Comment by mxnmnm** [comment]
Still happens

**Comment by Tschipp** [comment]
Was able to reproduce this (although not reliably) on a big modpack. Guess I'll have to start removing mods until I find the conflict...

**Comment by Tschipp** [comment]
Could you test with the latest version?

**Comment by mxnmnm** [comment]
Still happens
